research/specfiles/hosting-sources/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Hosting sources for Fedora
+authors: mfocko
+---
+
+:::tip Context
+
+tl;dr As long as Packit is deployed on the MP+, onboarding of new users is
+limited by the firewall rules that need to be explicitly allowed. Additionally
+firewall is mostly blocking `sync-release` actions specifically.
+
+This is outcome of [this](https://github.com/packit/packit-service/issues/2390)
+issue.
+
+:::
+
+All `SourceX` fields of the specfiles have been initially scraped by the @msuchy.
+This “research” provides the scripts that have been used to process the scraped
+sources.
+
```

Comment:
could you please include here some instructions on how the data can be obtained (how should be the script run)?

Reply:
Do you mean the script in this repo or the one that scrapes the specfiles?

Reply:
would it make sense to include both? It would be enough to mention here what files represent what.